examples/tutorial/go/Earthfile

```diff
@@ -18,3 +22,7 @@ part4:
 part5:
     COPY --dir ./part5 ./
     SAVE ARTIFACT ./part5
+
+part6:
+    COPY --dir ./part6 ./
+    SAVE ARTIFACT ./part6
```

Comment:
Can you also adjust our GH Actions CI script to execute part2 and part6 as part of the CI build? Look for a CI pipeline called [Tutorial](https://github.com/earthly/earthly/runs/5727227527?check_suite_focus=true)